diff -Nur plex86/conf/cooperative plex86-new/conf/cooperative
--- plex86/conf/cooperative	Fri Jan 19 18:07:03 2001
+++ plex86-new/conf/cooperative	Sat Mar  3 20:12:50 2001
@@ -16,7 +16,7 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: a
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
@@ -32,11 +32,12 @@
 
 # Replay IO events from a previous run.  For now this let's us
 # program the VGA adapter, without booting via the BIOS.
-plugin = ./plugins/misc/replay_io.so file=../misc/vga_io.log
+plugin = /usr/lib/plex86/plugins/misc/replay_io.so \
+  file=/usr/lib/plex86/misc/vga_io.log
 
 #====== Cooperative ==================================================
-plugin = ./plugins/loader/load-kernel.so \
-  file=../guest/cooperative/kernel \
+plugin = /usr/lib/plex86/plugins/loader/load-kernel.so \
+  file=/usr/lib/plex86/guest/cooperative/kernel \
   stack=36864
 
 ####################################################################
diff -Nur plex86/conf/freedos plex86-new/conf/freedos
--- plex86/conf/freedos	Mon Jan  1 17:16:31 2001
+++ plex86-new/conf/freedos	Sat Mar  3 20:13:12 2001
@@ -15,9 +15,9 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: c
-bochs-opt diskc: file="../../10M.FreeDOS", cyl=306, heads=4, spt=17
+#bochs-opt diskc: file="../../10M.FreeDOS", cyl=306, heads=4, spt=17
 #bochs-opt floppya: 1_44=../../1.44a, status=inserted
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
@@ -30,14 +30,14 @@
 #bochs-opt time0: 938581955
 bochs-init
 
-#plugin = ./plugins/write-cache/write-cache.so
+#plugin = /usr/lib/plex86/plugins/write-cache/write-cache.so
 
 
 # The system BIOS
-load-rom: file=../bios/BIOS-plex86-current, address=0xf0000
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
 
 # The VGA BIOS
-load-rom: file=../bios/VGABIOS-elpin-2.40, address=0xc0000
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
 
 
 ####################################################################
diff -Nur plex86/conf/grub plex86-new/conf/grub
--- plex86/conf/grub	Thu Dec 28 19:50:01 2000
+++ plex86-new/conf/grub	Sat Mar  3 20:13:25 2001
@@ -15,9 +15,9 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: c
-bochs-opt diskc: file=/home/kpl/plex86/grub_disk, cyl=256, heads=16, spt=63
+#bochs-opt diskc: file=/home/kpl/plex86/grub_disk, cyl=256, heads=16, spt=63
 #bochs-opt floppya: 1_44=/dev/fd0, status=inserted
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
@@ -32,10 +32,10 @@
 
 
 # The system BIOS
-load-rom: file=../bios/BIOS-plex86-current, address=0xf0000
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
 
 # The VGA BIOS
-load-rom: file=../bios/VGABIOS-elpin-2.40, address=0xc0000
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
 
 ####################################################################
 # Debugging settings
diff -Nur plex86/conf/linuxhd plex86-new/conf/linuxhd
--- plex86/conf/linuxhd	Thu Dec 28 19:50:01 2000
+++ plex86-new/conf/linuxhd	Sat Mar  3 20:13:37 2001
@@ -15,10 +15,10 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: c
 #bochs-opt floppya: 1_44=../../1.44a, status=inserted
-bochs-opt diskc: file=../../126M.linux, cyl=256, heads=16, spt=63
+#bochs-opt diskc: file=../../126M.linux, cyl=256, heads=16, spt=63
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
 bochs-opt vga_update_interval: 300000
@@ -32,10 +32,10 @@
 
 
 # The system BIOS
-load-rom: file=../bios/BIOS-plex86-current, address=0xf0000
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
 
 # The VGA BIOS
-load-rom: file=../bios/VGABIOS-elpin-2.40, address=0xc0000
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
 
 ####################################################################
 # Debugging settings
diff -Nur plex86/conf/msdos plex86-new/conf/msdos
--- plex86/conf/msdos	Thu Dec 28 19:50:01 2000
+++ plex86-new/conf/msdos	Sat Mar  3 20:13:47 2001
@@ -15,10 +15,10 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: a
 #bochs-opt diskc: file="../../10M.vga", cyl=306, heads=4, spt=17
-bochs-opt floppya: 1_44=../../1.44_dos622, status=inserted
+#bochs-opt floppya: 1_44=../../1.44_dos622, status=inserted
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
 bochs-opt vga_update_interval: 200000
@@ -32,10 +32,10 @@
 
 
 # The system BIOS
-load-rom: file=../bios/BIOS-plex86-current, address=0xf0000
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
 
 # The VGA BIOS
-load-rom: file=../bios/VGABIOS-elpin-2.40, address=0xc0000
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
 
 
 ####################################################################
diff -Nur plex86/conf/paging plex86-new/conf/paging
--- plex86/conf/paging	Sat Jan 13 04:51:38 2001
+++ plex86-new/conf/paging	Sat Mar  3 20:14:14 2001
@@ -16,7 +16,7 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: a
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
@@ -32,11 +32,12 @@
 
 # Replay IO events from a previous run.  For now this let's us
 # program the VGA adapter, without booting via the BIOS.
-plugin = ./plugins/misc/replay_io.so file=../misc/vga_io.log
+plugin = /usr/lib/plex86/plugins/misc/replay_io.so \
+  file=/usr/lib/plex86/misc/vga_io.log
 
 #====== Paging =======================================================
-plugin = ./plugins/loader/load-kernel.so \
-  file=../guest/paging/kernel \
+plugin = /usr/lib/plex86/plugins/loader/load-kernel.so \
+  file=/usr/lib/plex86/guest/paging/kernel \
   stack=36864
 
 
diff -Nur plex86/conf/pld plex86-new/conf/pld
--- plex86/conf/pld	Thu Jan  1 01:00:00 1970
+++ plex86-new/conf/pld	Sat Mar  3 20:21:57 2001
@@ -0,0 +1,61 @@
+####################################################################
+# This is an example plex86 configuration file
+####################################################################
+
+
+####################################################################
+# Memory settings
+####################################################################
+
+# number of megabytes for vm
+memory = 16
+
+
+####################################################################
+# Plugins
+####################################################################
+
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
+bochs-opt boot: a
+#bochs-opt floppya: 1_44=/home/users/krzyszcz/PLD/images/bootdisk.img, status=inserted
+#bochs-opt diskc: file=../../126M.linux, cyl=256, heads=16, spt=63
+bochs-opt keyboard_serial_delay: 200
+bochs-opt floppy_command_delay: 500
+bochs-opt vga_update_interval: 300000
+bochs-opt mouse: enabled=0
+bochs-opt private_colormap: enabled=0
+bochs-opt i440fxsupport: enabled=0
+bochs-opt newharddrivesupport: enabled=1
+bochs-opt ips: 1000000
+#bochs-opt time0: 938581955
+bochs-init
+
+
+# The system BIOS
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
+
+# The VGA BIOS
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
+
+####################################################################
+# Debugging settings
+####################################################################
+
+# syntax for disassembly
+db_syntax = at&t
+
+
+####################################################################
+# VM tweaks            
+####################################################################
+  
+# Maximum depth that the scan-before-execute code will try
+# to recursively prescan code in the current code page, by
+# following intra-page branches.
+prescan: depth = 3, ring3 = on
+
+
+
+####################################################################
+# end of config file
+####################################################################
diff -Nur plex86/conf/pragma plex86-new/conf/pragma
--- plex86/conf/pragma	Thu Dec 28 19:50:01 2000
+++ plex86-new/conf/pragma	Sat Mar  3 20:14:20 2001
@@ -15,9 +15,9 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: a
-bochs-opt floppya: 1_44=../../1.44_pragma386, status=inserted
+#bochs-opt floppya: 1_44=../../1.44_pragma386, status=inserted
 #bochs-opt diskc: file=../../126M.linux, cyl=256, heads=16, spt=63
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
@@ -32,10 +32,10 @@
 
 
 # The system BIOS
-load-rom: file=../bios/BIOS-plex86-current, address=0xf0000
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
 
 # The VGA BIOS
-load-rom: file=../bios/VGABIOS-elpin-2.40, address=0xc0000
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
 
 ####################################################################
 # Debugging settings
diff -Nur plex86/conf/preemptive plex86-new/conf/preemptive
--- plex86/conf/preemptive	Sat Jan 13 04:51:38 2001
+++ plex86-new/conf/preemptive	Sat Mar  3 20:06:38 2001
@@ -16,7 +16,7 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: a
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
@@ -32,31 +32,32 @@
 
 # Replay IO events from a previous run.  For now this let's us
 # program the VGA adapter, without booting via the BIOS.
-plugin = ./plugins/misc/replay_io.so file=../misc/vga_io.log
+plugin = /usr/lib/plex86/plugins/misc/replay_io.so \
+  file=/usr/lib/plex86/misc/vga_io.log
 
-plugin = ./plugins/loader/load-kernel.so \
-  file=../guest/preemptive/kernel \
+plugin = /usr/lib/plex86/plugins/loader/load-kernel.so \
+  file=/usr/lib/plex86/guest/preemptive/kernel \
   stack=36864
 
 #====== Cooperative ==================================================
-#plugin = ./plugins/loader/load-kernel.so \
-#  file=../guest/cooperative/kernel \
+#plugin = /usr/lib/plex86/plugins/loader/load-kernel.so \
+#  file=/usr/lib/plex86/guest/cooperative/kernel \
 #  stack=36864
 
 #====== Paging =======================================================
-#plugin = ./plugins/loader/load-kernel.so \
-#  file=../guest/paging/kernel \
+#plugin = /usr/lib/plex86/plugins/loader/load-kernel.so \
+#  file=/usr/lib/plex86/guest/paging/kernel \
 #  stack=36864
 
 #====== Linux ========================================================
-#plugin = ./plugins/loader/load-kernel.so \
+#plugin = /usr/lib/plex86/plugins/loader/load-kernel.so \
 #  file=../../linux.img \
 #  address=0x100000 \
 #  stack=36864 \
 #  rd=0x00e80000:../../initrd.img \
 #  hack=linux
 
-#plugin = ./plugins/ice/plugin-ice.so 3141
+#plugin = /usr/lib/plex86/plugins/ice/plugin-ice.so 3141
 
 
 ####################################################################
diff -Nur plex86/conf/qnx plex86-new/conf/qnx
--- plex86/conf/qnx	Thu Dec 28 19:50:01 2000
+++ plex86-new/conf/qnx	Sat Mar  3 20:16:31 2001
@@ -15,9 +15,9 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: a
-bochs-opt floppya: 1_44=../../1.44_qnx, status=inserted
+#bochs-opt floppya: 1_44=../../1.44_qnx, status=inserted
 #bochs-opt diskc: file=../../126M.linux, cyl=256, heads=16, spt=63
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
@@ -32,10 +32,10 @@
 
 
 # The system BIOS
-load-rom: file=../bios/BIOS-plex86-current, address=0xf0000
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
 
 # The VGA BIOS
-load-rom: file=../bios/VGABIOS-elpin-2.40, address=0xc0000
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
 
 ####################################################################
 # Debugging settings
diff -Nur plex86/conf/tmp plex86-new/conf/tmp
--- plex86/conf/tmp	Sat Jan 13 04:51:38 2001
+++ plex86-new/conf/tmp	Sat Mar  3 20:10:24 2001
@@ -16,7 +16,7 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: a
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
@@ -32,10 +32,11 @@
 
 # Replay IO events from a previous run.  For now this let's us
 # program the VGA adapter, without booting via the BIOS.
-plugin = ./plugins/misc/replay_io.so file=../misc/vga_io.log
+plugin = /usr/lib/plex86/plugins/misc/replay_io.so \
+  file=/usr/lib/plex86/misc/vga_io.log
 
 #====== Paging =======================================================
-plugin = ./plugins/loader/load-kernel.so \
+plugin = /usr/lib/plex86/plugins/loader/load-kernel.so \
   file=/tmp/kernel.elf \
   stack=36864
 
diff -Nur plex86/conf/tomsrtbt plex86-new/conf/tomsrtbt
--- plex86/conf/tomsrtbt	Thu Dec 28 19:50:01 2000
+++ plex86-new/conf/tomsrtbt	Sat Mar  3 20:10:56 2001
@@ -15,10 +15,10 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: a
 #bochs-opt diskc: file="../../10M.vga", cyl=306, heads=4, spt=17
-bochs-opt floppya: 1_44=../../tomsrtbt.raw, status=inserted
+#bochs-opt floppya: 1_44=../../tomsrtbt.raw, status=inserted
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
 bochs-opt vga_update_interval: 200000
@@ -32,10 +32,10 @@
 
 
 # The system BIOS
-load-rom: file=../bios/BIOS-plex86-current, address=0xf0000
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
 
 # The VGA BIOS
-load-rom: file=../bios/VGABIOS-elpin-2.40, address=0xc0000
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
 
 
 ####################################################################
diff -Nur plex86/conf/win95 plex86-new/conf/win95
--- plex86/conf/win95	Thu Dec 28 19:50:01 2000
+++ plex86-new/conf/win95	Sat Mar  3 20:11:23 2001
@@ -15,9 +15,9 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: c
-bochs-opt diskc: file="../../62M.980115", cyl=940, heads=8, spt=17
+#bochs-opt diskc: file="../../62M.980115", cyl=940, heads=8, spt=17
 #bochs-opt floppya: 1_44=../../1.44a, status=inserted
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
@@ -32,10 +32,10 @@
 
 
 # The system BIOS
-load-rom: file=../bios/BIOS-plex86-current, address=0xf0000
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
 
 # The VGA BIOS
-load-rom: file=../bios/VGABIOS-elpin-2.40, address=0xc0000
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
 
 
 ####################################################################
diff -Nur plex86/conf/winnt plex86-new/conf/winnt
--- plex86/conf/winnt	Thu Dec 28 19:50:01 2000
+++ plex86-new/conf/winnt	Sat Mar  3 20:16:44 2001
@@ -15,11 +15,11 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: c
-bochs-opt diskc: file=../../483M.NT40, cyl=1024, heads=15, spt=63
-bochs-opt cdromd: dev=/dev/cdrom, status=ejected
-bochs-opt floppya: 1_44=../../1.44a, status=inserted
+#bochs-opt diskc: file=../../483M.NT40, cyl=1024, heads=15, spt=63
+#bochs-opt cdromd: dev=/dev/cdrom, status=ejected
+#bochs-opt floppya: 1_44=../../1.44a, status=inserted
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
 bochs-opt vga_update_interval: 200000
@@ -33,10 +33,10 @@
 
 
 # The system BIOS
-load-rom: file=../bios/BIOS-plex86-current, address=0xf0000
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
 
 # The VGA BIOS
-load-rom: file=../bios/VGABIOS-elpin-2.40, address=0xc0000
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
 
 
 ####################################################################
diff -Nur plex86/conf/zealos plex86-new/conf/zealos
--- plex86/conf/zealos	Thu Dec 28 19:50:01 2000
+++ plex86-new/conf/zealos	Sat Mar  3 20:12:17 2001
@@ -15,9 +15,9 @@
 # Plugins
 ####################################################################
 
-plugin = ./plugins/bochs/plugin-bochs.so
+plugin = /usr/lib/plex86/plugins/bochs/plugin-bochs.so
 bochs-opt boot: a
-bochs-opt floppya: 1_44=/home/kpl/plex86/1.44_vmzealoz, status=inserted
+#bochs-opt floppya: 1_44=/home/kpl/plex86/1.44_vmzealoz, status=inserted
 bochs-opt keyboard_serial_delay: 200
 bochs-opt floppy_command_delay: 500
 bochs-opt vga_update_interval: 200000
@@ -31,10 +31,10 @@
 
 
 # The system BIOS
-load-rom: file=../bios/BIOS-plex86-current, address=0xf0000
+load-rom: file=/usr/lib/plex86/bios/BIOS-plex86-current, address=0xf0000
 
 # The VGA BIOS
-load-rom: file=../bios/VGABIOS-elpin-2.40, address=0xc0000
+load-rom: file=/usr/lib/plex86/bios/VGABIOS-elpin-2.40, address=0xc0000
 
 
 ####################################################################
